server/tests/api/test_flows.py

```diff
@@ -222,6 +222,18 @@ class TestCreateFlow:
         flow = await m.Flow.where(id=flow_id).first("description")
         assert flow.description == description
 
+    async def test_create_flow_intelligently_handles_scheduled_param_defaults(self):
+        a, b = prefect.Parameter("a"), prefect.Parameter("b", default=1)
+        clock = prefect.schedules.clocks.CronClock(
+            cron=f"* * * * *", parameter_defaults={"a": 1,}
```

Comment:
To align with my suggestion above, I think a second clock with defaults `{"a": 1, "b": 2}` would be interesting